package.json

```diff
@@ -28,7 +28,7 @@
   ],
   "dependencies": {
     "@actions/core": "1.10.0",
-    "shescape": "1.6.1"
+    "shescape": "1.5.10"
```

Comment:
Revert this prior to merging (NOTE: do this locally to ensure `package-lock.json` is also updated)

```diff
-    "shescape": "1.5.10"
+    "shescape": "1.6.1"
```